pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -327,39 +321,71 @@ func (l *listUsersQuery) expandIntersection(
 	pool.WithCancelOnError()
 	pool.WithMaxGoroutines(int(l.resolveNodeBreadthLimit))
 
-	intersectionFoundUsersChan := make(chan *openfgav1.User, 1)
+	wildcardKey := fmt.Sprintf("%s:*", req.GetUserFilters()[0].GetType())
```

Comment:
This should be a helper in the `pkg/tuple` package.

```go
package tuple
...
func TypedPublicWildcard(objectType string) string {
    return fmt.Sprintf("%s:%s", objectType, Wildcard)
```

Reply:
Addressed in https://github.com/openfga/openfga/pull/1490. Will pull-in that work once merged to main.